src/main/java/org/kumoricon/registration/utility/TillSessionController.java

```diff
@@ -79,8 +79,10 @@ public String closeTill(Model model,
             }
         } else if (action.equals("Close Till")) {
             tillSessionService.closeSessionForUser(currentUser, tillName);
-
-            return "redirect:/utility/till?msg=Till%20Session%20Closed";
+            String reportPrinterName = settingsService.getCurrentSettings().getReportPrinterName();
```

Comment:
The way it stands now, if the report fails to print it will just throw an error message (for example: "Error: Printer named '' does not exist and no default printer set"). 
There's not really anything for regular users to do at that point (and no way for them to reprint anyway). 

If it fails, I think I'd like it to just redirect the user to the next page with an error message ("warning: your report did not print automatically" or something like that.) but let them continue with whatever they were doing. We can always print the report on demand from the back anyway. But getting the person going off shift off the computer and whoever should be going on shift in the seat checking people in is most important

Reply:
Added to the PR